src/primitives.jl

```diff
@@ -54,3 +54,4 @@ include("primitives/conesurface.jl")
 include("primitives/frustum.jl")
 include("primitives/frustumsurface.jl")
 include("primitives/torus.jl")
+include("primitives/paraboloidsurface.jl")
```

Comment:
let's include the file after the cylindersurface.jl, all the methods you added happened after cylindersurface